### DISCUSSION TITLE
Disable timestamps in migration and query automated in v4

### DISCUSSION THREAD

**Discussion by the-prodigy** [adonisjs/core#1622] [Help]
Hi!

How can I disable to insert timestamps by default like at migration part and query part?

There is a solution that disables it:

```
 static get createdAtColumn () {
    return null;
  }

  static get updatedAtColumn () {
    return null;
  }
```

But then I have to paste that part into every model I create. How can I automate this?

**Comment by thetutlage** [comment]
Use class inheritance by creating a base model?

```ts
class BaseModel extends Model {
 static get createdAtColumn () {
    return null;
  }

  static get updatedAtColumn () {
    return null;
  }
}
```

And then other models can extend your base model

```js
const BaseModel = use('App/Models/BaseModel')
class User extends BaseModel {}
```

**Reply by the-prodigy** [reply]
Ah I saw an trait thats comes already with a file like NoTimestamps.

Can I use it like implement it and how?

**Reply by thetutlage** [reply]
Yes, you can use the trait or create a BaseModel. Depends upon what you prefer

**Reply by the-prodigy** [reply]
Can I edit the Lucid Core "Model" from adonisjs Like editing that and automating that process? 

Whats the best easiest way to automate this and not to manually use an trait or extend a different Model as the default Model?
